docs/DeveloperGuide.md

```diff
@@ -450,6 +450,21 @@ We propose to make the index error messages more specific and highlight to the u
 that index is wrong. For example, in the `enrol m/1 e/-1` input, we will show an error message to the user along 
 the lines of "The provided Event Index is not a non-zero unsigned integer."
 
+### Improve find criteria
```

Comment:
Suggestion to change this to Improve `findMember` and `findEvent` criteria

Reply:
Done!